helix-term/src/commands.rs

```diff
@@ -4646,9 +4646,11 @@ fn shell_prompt(cx: &mut Context, prompt: Cow<'static, str>, behavior: ShellBeha
     );
 }
 
-fn suspend(_cx: &mut Context) {
-    #[cfg(not(windows))]
-    signal_hook::low_level::raise(signal_hook::consts::signal::SIGTSTP).unwrap();
+fn suspend(cx: &mut Context) {
+    if cx.editor.suspend_enabled {
+        #[cfg(not(windows))]
+        signal_hook::low_level::raise(libc::SIGTSTP).unwrap();
+    }
```

Comment:
```suggestion
    if cfg!(not(windows)) && cx.editor.suspend_enabled {
        #[cfg(not(windows))]
        signal_hook::low_level::raise(libc::SIGTSTP).unwrap();
    } else {
        cx.editor.set_status("Suspend is not supported by this terminal").
    }
```

So people know what's up if C-z isn't working